### PR TITLE
 Modified color code of Navbar items

### PR DESCRIPTION
**Pull request by Subaharish**
Signed-off-by: Subaharish <subaharish2001@gmail.com>

Proposed Changes:

# New Ejs page has to be created

 -techweek
 -managementweek
 -culturalweek

### CODE REVIEW

**Comment by aravindvnair99** [comment]
@Subaharish I'm moving this to draft. Once all of #13 is completed, you can request for review.

**Review by VinayakTekade — DISMISSED** [review]

File: functions/views/contact.ejs [hunk 1]
```diff
@@ -1,112 +1,141 @@
...
+						<div class="col-lg-7">
+							<div
+								class="contact-info__item-left wow fadeInRight"
+							>
+								<div class="fillin-form">
+									<form action="#">
+										<div class="fillin-form__fields">
+											<input
+												type="text"
+												placeholder="Name"
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/contact.ejs:101]: Will resolve this issue, when we work on Contact Page

File: functions/views/contact.ejs [hunk 2]
```diff
@@ -1,112 +1,141 @@
...
+											<input
+												type="email"
+												placeholder="Email"
+											/>
+										</div>
+
+										<textarea
+											placeholder="Message"
+											cols="30"
+											rows="10"
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/contact.ejs:112]: Will resolve this issue, when we work on Contact Page

File: functions/views/contact.ejs [hunk 3]
```diff
@@ -1,112 +1,141 @@
...
+												class="hand-draw"
+											>
+												<path
+													class="hand-med-hover"
+													d="M33.2,31.2C53.8,14.1,78.4,9.5,96.9,19.4c12.2,6.5,26.7,21.9,23.9,37.8c-3.9,22.2-39.1,37.1-65.6,34.5c-8.8-0.9-17.6-3.8-25.3-8.2C16.7,76,4.5,63.1,4.4,47.2C4.3,32.4,15,19.3,27.2,12.1c3.6-2.1,7.5-3.9,11.5-5.3c15.2-5.5,27.7-2.5,32.7-1.3c3.6,0.9,18,4.9,29.9,19c5.4,6.3,11.2,13.1,12.5,24.5c0.3,2.6,2,20.2-10.5,34.1C93.9,93.4,82,94,70,94.7c-9.9,0.6-25.8,1.5-39.3-10.1c-2.3-2-20.5-18.1-16-38.8c2.4-10.8,10-18.2,12.4-20.5c6.8-6.6,13.6-8.9,19.7-11C52,12.7,63,9.1,76.3,11.8c5,1,14.8,3.1,23.4,11.9c2.4,2.5,9.2,9.7,11.6,22c0.8,4.2,2.2,11.4-0.4,19.1c-4.9,14.2-20.9,19.7-28.3,22.4c-6,2.1-18,6.4-31.1,1.9c-16.1-5.4-33.7-23.9-28.8-44.2C25.5,33.2,35,24,44.6,19.9c14.3-6.2,26.1,0,28,1"
+												/>
+											</svg>
+											<input
+												class="moving-btn"
+												type="submit"
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/contact.ejs:126]: Will resolve this issue, when we work on Contact Page

File: functions/views/template/blogSingle.ejs [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+										>
+									</div>
+								</div>
+							</div>
+						</div>
+						<h5 class="comment__leave-comment">Leave A Comment</h5>
+						<div class="fillin-form">
+							<form action="#">
+								<div class="fillin-form__fields">
+									<input type="text" placeholder="Name" />
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/template/blogSingle.ejs:275]: This issue is not necessary as this is a template file.

File: functions/views/template/blogSingle.ejs [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+									<input type="email" placeholder="Email" />
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/template/blogSingle.ejs:276]: This issue is not necessary as this is a template file.

File: functions/views/template/blogSingle.ejs [hunk 6]
```diff
@@ -0,0 +1,306 @@
...
+								</div>
+
+								<textarea
+									placeholder="Message"
+									cols="30"
+									rows="10"
+								></textarea>
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/template/blogSingle.ejs:283]: This issue is not necessary as this is a template file.

File: functions/views/template/blogSingle.ejs [hunk 7]
```diff
@@ -0,0 +1,306 @@
...
+									<svg
+										viewBox="4 0 118 98.6"
+										class="hand-draw"
+									>
+										<path
+											class="hand-med-hover"
+											d="M33.2,31.2C53.8,14.1,78.4,9.5,96.9,19.4c12.2,6.5,26.7,21.9,23.9,37.8c-3.9,22.2-39.1,37.1-65.6,34.5c-8.8-0.9-17.6-3.8-25.3-8.2C16.7,76,4.5,63.1,4.4,47.2C4.3,32.4,15,19.3,27.2,12.1c3.6-2.1,7.5-3.9,11.5-5.3c15.2-5.5,27.7-2.5,32.7-1.3c3.6,0.9,18,4.9,29.9,19c5.4,6.3,11.2,13.1,12.5,24.5c0.3,2.6,2,20.2-10.5,34.1C93.9,93.4,82,94,70,94.7c-9.9,0.6-25.8,1.5-39.3-10.1c-2.3-2-20.5-18.1-16-38.8c2.4-10.8,10-18.2,12.4-20.5c6.8-6.6,13.6-8.9,19.7-11C52,12.7,63,9.1,76.3,11.8c5,1,14.8,3.1,23.4,11.9c2.4,2.5,9.2,9.7,11.6,22c0.8,4.2,2.2,11.4-0.4,19.1c-4.9,14.2-20.9,19.7-28.3,22.4c-6,2.1-18,6.4-31.1,1.9c-16.1-5.4-33.7-23.9-28.8-44.2C25.5,33.2,35,24,44.6,19.9c14.3-6.2,26.1,0,28,1"
+										/>
+									</svg>
+									<input class="moving-btn" type="submit" />
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/template/blogSingle.ejs:294]: This issue is not necessary as this is a template file.

**Review by aravindvnair99 — APPROVED** [review]

**Review by aravindvnair99 — APPROVED** [review]

**Pull request merged**